frame/elections-phragmen/src/lib.rs

```diff
@@ -779,7 +795,9 @@ impl<T: Config> Pallet<T> {
 				// defensive-only: Members and runners-up are disjoint. This will always be err and
 				// give us an index to insert.
 				if let Err(index) = members.binary_search_by(|m| m.who.cmp(&next_best.who)) {
-					members.insert(index, next_best.clone());
+					members
+						.try_insert(index, next_best.clone())
+						.expect("Cannot accept more than DesiredMembers.");
```

Comment:
Maybe we can use `log::error` just like we do a few lines down from here to not complicate the return types. And also this shouldn't really ever happen because of the reason I have put inside the error message.